scripts/build_test.py

```diff
@@ -733,13 +741,15 @@ def test_compiled_js_dir_validation(self):
         with python_utils.open_file(build.TSCONFIG_FILEPATH, 'r') as f:
             config_data = json.load(f)
             out_dir = os.path.join(config_data['compilerOptions']['outDir'], '')
+        error_message = ('COMPILED_JS_DIR: %s does not match the output '
+                         'directory in %s: %s' % (
+                             MOCK_COMPILED_JS_DIR, build.TSCONFIG_FILEPATH,
+                             out_dir))
+        error_message = re.escape(error_message)
         with self.assertRaisesRegexp(
             Exception,
-            'COMPILED_JS_DIR: %s does not match the output directory '
-            'in %s: %s' % (
-                MOCK_COMPILED_JS_DIR, build.TSCONFIG_FILEPATH,
-                out_dir)), self.swap(
-                    build, 'COMPILED_JS_DIR', MOCK_COMPILED_JS_DIR):
+            error_message), self.swap(
```

Comment:
I think you can wrap this and the previous line to be at the end of line 749.

Reply:
Updated